appveyor.yml

```diff
@@ -8,8 +8,13 @@ configuration:
   - LLVM 3.9.0
 
 environment:
-  DVersion: 2.071.0
-  DC: dmd
+  matrix:
+  - DC: dmd
+    DVersion: 2.076.0
+    arch: x86
+  - DC: dmd
+    DVersion: 2.076.0
+    arch: x86_64
```

Comment:
I've removed the usage of `arch`. The appveyor script now uses the `PLATFORM` environment variable to identify the architecture. A build matrix is automatically created when `platform` [1] is used, which will automatically set the `PLATFORM` environment variable.

[1] https://github.com/jacob-carlborg/dstep/blob/master/appveyor.yml#L1

Reply:
Fixed.